06_PDDE_Checker.R

```diff
@@ -277,14 +277,16 @@ overlapping_ce_participation <- CEParticipation %>%
   mutate(ParticipationPeriod =
            interval(
              CEParticipationStatusStartDate,
-             coalesce(CEParticipationStatusEndDate, meta_HUDCSV_Export_End)),
+             coalesce(CEParticipationStatusEndDate, no_end_date)),
          PreviousParticipationPeriod = 
            interval(
              PreviousCEStart,
-             coalesce(PreviousCEEnd, meta_HUDCSV_Export_End)
+             coalesce(PreviousCEEnd, no_end_date)
            ),
-         OverlapYN = int_overlaps(ParticipationPeriod, PreviousParticipationPeriod),
-         Detail = paste(
+         OverlapYN = int_overlaps(ParticipationPeriod, PreviousParticipationPeriod)
+  ) %>%
+  filter(OverlapYN) %>%
```

Comment:
couldn't find this in the tidy style guide, but in the rest of Eva, we keep with the "sentence" structure inside filter()s, so like OverlapYN == TRUE would align with the rest of Eva syntax

Reply:
okay I'll make the change.